djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor_plugins/cmsdialog/plugin.js

```diff
@@ -1,3 +1,7 @@
+(function ($) {
+if (CKEDITOR && CKEDITOR.plugins && CKEDITOR.plugins.registered && CKEDITOR.plugins.registered.cmsdialog) {
+    return;
+}
```

Comment:
ignore this one, this is an override of the ckeditor's own `dialog` plugin, better to keep unindented for easier diffs when upgrading